src/cms/templates/dashboard/admin_dashboard.html

```diff
@@ -1,5 +1,32 @@
 {% extends "_base.html" %}
 {% load i18n %}
 {% block content %}
-    Admin Dashboard
-{% endblock %}
\ No newline at end of file
+{% load static %}
+{% load widget_tweaks %}
+   
+   <h2>{% trans 'Technical Feedback' %}</h2>
+
+<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+    <tr class="border-b border-solid border-gray-200">
+        <th class="text-sm text-left uppercase py-3">{% trans 'Language' %}</th>
+        <th class="text-sm text-left uppercase py-3">{% trans 'Comment' %}</th>
+        <th class="text-sm text-left uppercase py-3">{% trans 'Rating' %}</th>
+        <th class="text-sm text-right uppercase py-3 pr-2">{% trans 'Date' %}</th>
+    </tr>
+  {% if not  admin_feedback_exists %}
+    <tr><td colspan="6" class="px-2 py-3">{% trans 'No Feedback available yet.' %} </td></tr>
+    
+  {% else%} 
+    {% for feedback in all_feedback %}
+     <tr>
+        
+       <td class="pl-2 py-3">{{ feedback.language.translated_name }}</td>
+        <td>{{ feedback.comment }}</td>
+        <td>{{ feedback.get_emotion_display }}</td>
+        <td class="pr-2 text-right">{{ feedback.created_date }}</td> 
+    </tr>
+    {% endfor %}
+  {% endif%}
```

Comment:
```suggestion
    {% for feedback in all_feedback %}
     <tr>
         <td class="pl-2 py-3">{{ feedback.language.translated_name }}</td>
          <td>{{ feedback.comment }}</td>
          <td>{{ feedback.get_emotion_display }}</td>
          <td class="pr-2 text-right">{{ feedback.created_date }}</td> 
    </tr>
    {% empty %}
    <tr><td colspan="6" class="px-2 py-3">{% trans 'No Feedback available yet.' %} </td></tr>
    {% endfor %}
```
If you have only one Queryset, you don't need the additional variable `admin_feedback_exists`. Just use the tag `{% empty %}`.
Also, please keep the indentation levels consistent at 4 spaces.

Reply:
I make changes.